src/piadm.sh

```diff
@@ -911,14 +952,75 @@ update_CN() {
 	update_boot_sectors "$pool" "$bootfs"
 }
 
+bringup_HN() {
+	# One last reality check...
+	if [[ "$pool" == "$TRITON_HN_BOOTPOOL" ]]; then
+		err "Enabling pool we just booted from."
```

Comment:
Because of the TRITON_HN_BOOTPOOL knowledge we can state we already booted it too.  But you're right we should call out $pool:

`err "Pool $pool is already bootable, and we just booted it."`